src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -308,10 +308,12 @@ public int compare(FeedbackResponseAttributes o1, FeedbackResponseAttributes o2)
                 return order;
             }
 
-            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty() ? getNameForEmail(o1.recipientEmail)
-                                                                                    : getTeamNameForEmail(o1.recipientEmail);
-            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty() ? getNameForEmail(o2.recipientEmail)
-                                                                                    : getTeamNameForEmail(o2.recipientEmail);
+            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o1.recipientEmail)
+                                 : getTeamNameForEmail(o1.recipientEmail);
+            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o2.recipientEmail)
+                                 : getTeamNameForEmail(o2.recipientEmail);
```

Comment:
This is one of them.
![screen shot 2016-05-31 at 12 53 57 pm](https://cloud.githubusercontent.com/assets/7261051/15663539/c7755fee-272e-11e6-9e7f-a5f464ce7f82.png)